doc/decisions/STEPS.md

```diff
@@ -0,0 +1,86 @@
+# Steps
+
+This document describes all steps a decision can run through.
+
+```mermaid
+flowchart LR
+    s((Start)) --> Drafts --> In_Discussion --> In_Progress --> Decided --> Partially_Implemented --> Implemented
+
+    %% Shortcuts:
+    s --> Decided
+    Decided --> Implemented
+
+    %% Dead ends:
+    s --x Rejected
+    s --o Delayed
+    Drafts --x Rejected
+    Drafts --o Delayed
+    In_Discussion --x Rejected
+    In_Discussion --o Delayed
+    In_Progress --x Rejected
+    In_Progress --o Delayed
+```
+
+## Drafts
+
+> This step is recommended if the problem is not yet clear.
```

Comment:
"Clear" to whom? In most cases the problem will be clear to the decision author.

Right now it's not at all clear when this step should be taken. With the extra sentence I suggested above, it would be clearer IMO. A decision always starts a "Draft". It can be merged as a "Draft", but it can also progress to "In Discussion" or even "Decided" before the merge.